subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/BuildVerdictPlugin.kt

```diff
@@ -29,12 +29,21 @@ class BuildVerdictPlugin : Plugin<ProjectInternal> {
 
         if (project.pluginIsEnabled) {
             val extension = project.extensions.create<BuildVerdictPluginExtension>("buildVerdict")
+            val outputDir = extension.outputDir
             val services = BuildVerdictPluginServices()
             project.gradle.addListener(services.gradleTaskExecutionListener())
             project.gradle.addLogEventListener(services.gradleLogEventListener())
+            val configurationListener = services.gradleConfigurationListener(outputDir, project.ciLogger)
+            project.gradle.addBuildListener(configurationListener)
```

Comment:
Important change If you want to `removeListener` you must `addListener`
`buildFinished` action won't be removed